src/log.py

```diff
@@ -0,0 +1,42 @@
+import logging
+from pathlib import Path
+
+
+def setup_logger(log_file: Path = None, level=logging.INFO) -> logging.Logger:
```

Comment:
parameterization for loggers (levels, whether to log to file) this would also be something to put in your config :)

Reply:
Good callout! I'll put it in the config.yaml